Editor/Scripts/Utilities/MeshsyncAnalytics.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+using UnityEditor;
+
+using UnityEngine;
+
+namespace Unity.MeshSync.Editor.Analytics {
```

Comment:
Similarly here, I think there should be an Analytics folder under Scripts to contain this file